doc/faq.rst

```diff
@@ -6,267 +6,520 @@
+    :ref:`sphx_glr_examples_40_advanced_example_interpretable_models.py`.
 
-Auto-sklearn wraps scikit-learn and therefore inherits its parallelism implementation. In short,
-scikit-learn uses two modes of parallelizing computations:
+.. collapse:: <b>How can I turn off data preprocessing?</b>
 
-1. By using joblib to distribute independent function calls on multiple cores.
-2. By using lower level libraries such as OpenMP and numpy to distribute more fine-grained
-   computation.
+    Data preprocessing includes One-Hot encoding of categorical features, imputation
+    of missing values and the normalization of features or samples. These ensure that
+    the data the gets to the sklearn models is well formed and can be used for
+    training models.
 
-This means that Auto-sklearn can use more resources than expected by the user. For technical
-reasons we can only control the 1st way of parallel execution, but not the 2nd. Thus, the user
-needs to make sure that the lower level parallelization libraries only use as many cores as
-allocated (on a laptop or workstation running a single copy of Auto-sklearn it can be fine to not
-adjust this, but when using a compute cluster it is necessary to align the parallelism setting
-with the number of requested CPUs). This can be done by setting the following environment
-variables: ``MKL_NUM_THREADS``, ``OPENBLAS_NUM_THREADS``, ``BLIS_NUM_THREADS`` and
-``OMP_NUM_THREADS``.
+    While this is necessary in general, if you'd like to disable this step, please
+    refer to this :ref:`example <sphx_glr_examples_80_extending_example_extending_data_preprocessor.py>`.
 
-More details can be found in the `scikit-learn docs <https://scikit-learn.org/stable/computing/parallelism.html?highlight=joblib#parallelism>`_.
+.. collapse:: <b>How can I turn off feature preprocessing?</b>
+
+    Feature preprocessing is a single transformer which implements for example feature
+    selection or transformation of features into a different space (i.e. PCA).
+
+    This can be turned off by setting
+    ``include={'feature_preprocessor'=["no_preprocessing"]}`` as shown in the example above.
+
+.. collapse:: <b>Will non-scikit-learn models be added to Auto-sklearn?</b>
+
+    The short answer: no.
+
+    The long answer answer is a bit more nuanced: maintaining Auto-sklearn requires a lot of time and
+    effort, which would grow even larger when depending on more libraries. Also, adding more
+    libraries would require us to generate meta-data more often. Lastly, having more choices does not
+    guarantee a better performance for most users as having more choices demands a longer search for
+    good models and can lead to more overfitting.
+
+    Nevertheless, everyone can still add their favorite model to Auto-sklearn's search space by
+    following the `examples on how to extend Auto-sklearn
+    <https://automl.github.io/auto-sklearn/master/examples/index.html#extension-examples>`_.
+
```

Comment:
I'm not sure distributing this over two lines will work, would need to check the generated docs.